locales/en.yml

```diff
@@ -1,4 +1,27 @@
 home:
   categories:
     newestTools: Newest tools
-
+    favoriteTools: 'Your favorite tools'
+    allTools: 'All the tools'
+  subtitle: 'Handy tools for developers'
+  toggleMenu: 'Toggle menu'
+  home: Home
+  uiLib: 'UI Lib'
+  buyMeACoffee: 'Buy me a coffee'
+  follow:
+    title: 'You like it-tools?'
+    p1: 'Give us a star on'
+    githubRepository: 'IT-Tools'''' GitHub repository'
```

Comment:
What's this weird syntax  `''''` ? 🤔

Reply:
errors in string conversion, fix and submit